game.py

```diff
@@ -12,6 +13,11 @@
 BIRD_BOOST = 20
 FONT_SIZE = 48
 
+UPDATE_SPRITE_EVENT = pygame.USEREVENT + 1
+ASSETS = "./assets"
+
+
+BIRD_IMAGES = [pygame.transform.scale(pygame.image.load(os.path.abspath(f"{ASSETS}/{file}")), (BIRD_SIZE, BIRD_SIZE)) for file in os.listdir(ASSETS)]
```

Comment:
Prefer `pathlib` over `os` for filesystem operations. In this case, something like the `glob` method on `pathlib.Path` could probably work.